data/pigui/modules/settings-window.lua

```diff
@@ -311,45 +273,57 @@ local function showVideoOptions()
 end
 
 local captureBindingWindow
+local bindState = nil -- state, to capture the key combination
 captureBindingWindow = ModalWindow.New("CaptureBinding", function()
 	local info = keyCaptureBind
 
-	ui.text(localize_binding_id(info.id))
+	ui.text(bindManager.localizeBindingId(info.id))
 	ui.text(lui.PRESS_A_KEY_OR_CONTROLLER_BUTTON)
 
 	if info.type == 'Action' then
 		local desc = keyCaptureNum == 1 and info.binding or info.binding2
-		ui.text(desc.enabled and get_chord_desc(desc) or '<None>')
+		ui.text(desc.enabled and bindManager.getChordDesc(desc) or '<None>')
```

Comment:
The `<None>` text should probably be localized - that's my bad for forgetting to do it when I introduced the bindings window.

Reply:
Done!